bot/main.py

```diff
@@ -324,25 +326,84 @@ def send_logs(update: Update, _: CallbackContext) -> None:
     update.effective_message.reply_text(text=klippy.get_versions_info(), disable_notification=notifier.silent_commands, quote=True)
     logs_list: List[Union[InputMediaAudio, InputMediaDocument, InputMediaPhoto, InputMediaVideo]] = []
-    if Path(configWrap.bot_config.log_file).exists():
-        with open(configWrap.bot_config.log_file, "rb") as fh:
-            logs_list.append(InputMediaDocument(fh.read(), filename="telegram.log"))
-    if Path(f"{configWrap.bot_config.log_path}/klippy.log").exists():
-        with open(f"{configWrap.bot_config.log_path}/klippy.log", "rb") as fh:
-            logs_list.append(InputMediaDocument(fh.read(), filename="klippy.log"))
-    if Path(f"{configWrap.bot_config.log_path}/moonraker.log").exists():
-        with open(f"{configWrap.bot_config.log_path}/moonraker.log", "rb") as fh:
-            logs_list.append(InputMediaDocument(fh.read(), filename="moonraker.log"))
+    log_files_names: List[str] = ["telegram.log", "klippy.log", "moonraker.log", "crowsnest.log"]
+    for log_file in log_files_names:
+        if Path(f"{configWrap.bot_config.log_path}/{log_file}").exists():
+            with open(f"{configWrap.bot_config.log_path}/{log_file}", "rb") as fh:
+                logs_list.append(InputMediaDocument(fh.read(), caption="Upload logs to analyzer /upload_logs", filename=log_file))
+
     if logs_list:
         update.effective_message.reply_media_group(logs_list, disable_notification=notifier.silent_commands, quote=True)
     else:
         update.effective_message.reply_text(
-            text="No logs found in log_path",
+            text=f"No logs found in log_path `{configWrap.bot_config.log_path}`",
             disable_notification=notifier.silent_commands,
             quote=True,
         )
 
 
+def upload_logs(update: Update, _: CallbackContext) -> None:
+    if update.effective_message is None or update.effective_message.bot is None:
+        logger.warning("Undefined effective message or bot")
+        return
+
+    if Path(f"{configWrap.bot_config.log_path}/dmesg.txt").exists():
+        Path(f"{configWrap.bot_config.log_path}/dmesg.txt").unlink()
+
+    subprocess.run(f"dmesg -T > {configWrap.bot_config.log_path}/dmesg.txt", shell=True, executable="/bin/bash", check=False)
+
+    if Path(f"{configWrap.bot_config.log_path}/debug.txt").exists():
+        Path(f"{configWrap.bot_config.log_path}/debug.txt").unlink()
+
+    commands = [
+        "lsb_release -a",
+        "uname -a",
+        "find /dev/serial",
+        "find /dev/v4l",
+        "free -h",
+        "df -h",
+        "lsusb",
+        "systemctl status KlipperScreen",
+        "systemctl status klipper-mcu",
+        "ip --details --statistics link show dev can0",
+    ]
+    for command in commands:
+        subprocess.run(f"{command} >> {configWrap.bot_config.log_path}/debug.txt", shell=True, executable="/bin/bash", check=False)
```

Comment:
please add command newself and empty newline after:

command
command output
empty newline